openreview/profile/process/dblp_record_process.js

```diff
@@ -8,9 +8,16 @@ async function process(client, edit, invitation) {
   if (authorids) {
     note.content.authorids.value = note.content.authorids.value.map((authorid, index) => authorids[index] || authorid);
   }
-  
-  console.log('note: ' + JSON.stringify(note));
-  
+
+  const html = note.content.html?.value;
+
+  if (html) {
```

Comment:
this is great! should we do the same to extract the PDF link?

Reply:
is it necessary to extract the PDF link?
from what i understand it's just the redirected url of html, not the link to a .pdf file

Reply:
I'm not sure but Andrew wants to complete the PDF value with the link to the PDF not to a webpage.